Olivaw

NicoRobertIn opened the pull request:
A few deletions in order to isolate the whole test process into a separate repository https://github.com/Wimmics/olivaw

andreiciortea reviewed (approved):
Very cool work, thanks @NicoRobertIn!

NicoRobertIn commented:
As requested by @andreiciortea I updated the workflow so that it only targets main, and removed the badge updater (on branch publication) since it was not necessary anymore
